Added links between pages

MarkPuchalaII opened the pull request:
Created links between each respective tutorial page to streamline the installation process a bit more.

MarkPuchalaII committed:
Added links between pages

Created links between each respective tutorial page to streamline the installation process a bit more.

MarkPuchalaII commented:
Not sure I did this right. @jhsu802701 You reading this?

jhsu802701 commented:
* Create a .gitignore file and add .DS_Store to it.  (At the moment, your pull request adds this file.)
* In chapter 2, add a link to chapter 3.
* In chapter 4, add a link to chapter 5.
* In chapter 5, add a link to chapter 6.
* In chapter 6, add a link to chapter 7.
* In chapter 7, add a link to chapter 8.

MarkPuchalaII committed:
Fixed formatting for the links

I noticed the links were showing the literal bolding asterisks because I put in too many spaces.
Removed those spaces, now.

MarkPuchalaII commented:
All of this was done in the initial pull request. 
Need anything else?
